test/recipe_test.rb

```diff
@@ -0,0 +1,56 @@
+require './lib/ingredient'
+require './lib/recipe'
+require 'pry'
+require 'minitest/autorun'
+
+class RecipeTest < Minitest::Test
+  def setup
+    @ingredient1 = Ingredient.new({name: "Cheese", unit: "C", calories: 100})
+    @ingredient2 = Ingredient.new({name: "Macaroni", unit: "oz", calories: 30})
+    @ingredient3 = Ingredient.new({name: "Ground Beef", unit: "oz", calories: 100})
+    @ingredient4 = Ingredient.new({name: "Bun", unit: "g", calories: 75})
+    @recipe1 = Recipe.new("Mac and Cheese")
+    @recipe2 = Recipe.new("Cheese Burger")
+  end
+  def test_it_is
+    assert_instance_of Recipe, @recipe1
+  end
+
+  def test_it_has_attributes
+    expected = {}
+    assert_equal "Mac and Cheese", @recipe1.name
+    assert_equal expected, @recipe1.ingredients_required
```

Comment:
Instead of the variable here, you could have put the hash wrapped in parenthesis in this line.